Description: This patch removes all references to `cust.img` from the flashing scripts, partition configurations, and related files.

diff --git a/flash_all.bat b/flash_all.bat
index 0000000..0000000 100755
--- a/flash_all.bat
+++ b/flash_all.bat
@@ -77,7 +77,6 @@ fastboot %* flash metadata %~dp0images\metadata.img || @echo "Flash metadata err
 ::fastboot %* flash persist %~dp0images\persist.img || @echo "Flash persist error" && exit /B 1
 fastboot %* flash rescue %~dp0images\rescue.img || @echo "Flash rescue error" && exit /B 1
 fastboot %* flash super %~dp0images\super.img || @echo "Flash super error" && exit /B 1
-fastboot %* flash cust %~dp0images\cust.img || @echo "Flash super error" && exit /B 1
 fastboot %* erase userdata
 fastboot %* flash userdata %~dp0images\userdata.img || @echo "Flash userdata error" && exit /B 1
 
diff --git a/flash_all.sh b/flash_all.sh
index 0000000..0000000 100755
--- a/flash_all.sh
+++ b/flash_all.sh
@@ -79,7 +79,6 @@ fastboot $* flash metadata   `dirname $0`/images/metadata.img
 #fastboot $* flash persist   `dirname $0`/images/persist.img
 fastboot $* flash rescue   `dirname $0`/images/rescue.img
 fastboot $* flash super   `dirname $0`/images/super.img
-fastboot $* flash cust `dirname $0`/images/cust.img
 fastboot $* erase userdata
 fastboot $* flash userdata   `dirname $0`/images/userdata.img
 
diff --git a/flash_all_except_data_storage.bat b/flash_all_except_data_storage.bat
index 0000000..0000000 100755
--- a/flash_all_except_data_storage.bat
+++ b/flash_all_except_data_storage.bat
@@ -74,7 +74,6 @@ fastboot %* flash vbmeta_system_b %~dp0images\vbmeta_system.img || @echo "Flash
 ::fastboot %* flash persist %~dp0images\persist.img || @echo "Flash persist error" && exit /B 1
 fastboot %* flash rescue %~dp0images\rescue.img || @echo "Flash rescue error" && exit /B 1
 fastboot %* flash super %~dp0images\super.img || @echo "Flash super error" && exit /B 1
-fastboot %* flash cust %~dp0images\cust.img || @echo "Flash super error" && exit /B 1
 
 fastboot %* getvar unlocked 2>&1 | findstr /r "yes" 2>&1 && fastboot %* set_active a
 fastboot %* oem erase-vb-index
diff --git a/flash_all_except_data_storage.sh b/flash_all_except_data_storage.sh
index 0000000..0000000 100755
--- a/flash_all_except_data_storage.sh
+++ b/flash_all_except_data_storage.sh
@@ -78,7 +78,6 @@ fastboot $* flash vbmeta_system_b   `dirname $0`/images/vbmeta_system.img
 #fastboot $* flash persist   `dirname $0`/images/persist.img
 fastboot $* flash rescue   `dirname $0`/images/rescue.img
 fastboot $* flash super   `dirname $0`/images/super.img
-fastboot $* flash cust `dirname $0`/images/cust.img
 
 fastboot $* set_active a
 fastboot $* oem erase-vb-index
diff --git a/flash_all_lock.bat b/flash_all_lock.bat
index 0000000..0000000 100755
--- a/flash_all_lock.bat
+++ b/flash_all_lock.bat
@@ -77,7 +77,6 @@ fastboot %* flash metadata %~dp0images\metadata.img || @echo "Flash metadata err
 ::fastboot %* flash persist %~dp0images\persist.img || @echo "Flash persist error" && exit /B 1
 fastboot %* flash rescue %~dp0images\rescue.img || @echo "Flash rescue error" && exit /B 1
 fastboot %* flash super %~dp0images\super.img || @echo "Flash super error" && exit /B 1
-fastboot %* flash cust %~dp0images\cust.img || @echo "Flash super error" && exit /B 1
 fastboot %* erase userdata
 fastboot %* flash userdata %~dp0images\userdata.img || @echo "Flash userdata error" && exit /B 1
 
diff --git a/flash_all_lock.sh b/flash_all_lock.sh
index 0000000..0000000 100755
--- a/flash_all_lock.sh
+++ b/flash_all_lock.sh
@@ -79,7 +79,6 @@ fastboot $* flash metadata   `dirname $0`/images/metadata.img
 #fastboot $* flash persist   `dirname $0`/images/persist.img
 fastboot $* flash rescue   `dirname $0`/images/rescue.img
 fastboot $* flash super   `dirname $0`/images/super.img
-fastboot $* flash cust `dirname $0`/images/cust.img
 fastboot $* erase userdata
 fastboot $* flash userdata   `dirname $0`/images/userdata.img
 
diff --git a/flash_gen_crc_list.py b/flash_gen_crc_list.py
index 0000000..0000000 100755
--- a/flash_gen_crc_list.py
+++ b/flash_gen_crc_list.py
@@ -7,7 +7,6 @@ import binascii
 sparse_file_list = {
     "super":"super.img",
     "userdata":"userdata.img",
-    "cust":"cust.img",
     "metadata":"metadata.img",
     "rescue":"rescue.img",
 }
diff --git a/images/partition.xml b/images/partition.xml
index 0000000..0000000 100755
--- a/images/partition.xml
+++ b/images/partition.xml
@@ -20,8 +20,7 @@
         <partition label="keystore" size_in_kb="512" type="DE7D4029-0F5B-41C8-AE7E-F6C023A02B33" bootable="false" readonly="false" filename="" />
         <partition label="frp" size_in_kb="512" type="91B72D4D-71E0-4CBF-9B8E-236381CFF17A" bootable="false" readonly="false" filename=""/>
         <partition label="minidump" size_in_kb="131072" type="71D24153-E4E2-4275-815C-08E10959803B" bootable="false" readonly="false" filename=""/>
-        <partition label="cust" size_in_kb="2097152" type="0FC63DAF-8483-4772-8E79-3D69D8477DE4" bootable="false" readonly="true" filename="cust.img" sparse="true" />
-	<partition label="super" size_in_kb="8912896" type="89A12DE1-5E41-4CB3-8B4C-B1441EB5DA38" bootable="false" readonly="false" filename="super.img" sparse="true"/>
+      <partition label="super" size_in_kb="8912896" type="89A12DE1-5E41-4CB3-8B4C-B1441EB5DA38" bootable="false" readonly="false" filename="super.img" sparse="true"/>
         <partition label="vbmeta_system_a" size_in_kb="64" type="1344859D-3A6A-4C14-A316-9E696B3A5400" bootable="false" readonly="true" filename="vbmeta_system.img"/>
         <partition label="vbmeta_system_b" size_in_kb="64" type="FE3AB853-5B66-4D4A-BF85-8D90AF1C2C4A" bootable="false" readonly="true" filename="vbmeta_system.img"/>
 		<partition label="metadata" size_in_kb="16384" type=" 988A98C9-2910-4123-AAEC-1CF6B1BC28F9" bootable="false" readonly="false" filename="metadata.img" sparse="true"/>
diff --git a/images/rawprogram0.xml b/images/rawprogram0.xml
index 0000000..0000000 100644
--- a/images/rawprogram0.xml
+++ b/images/rawprogram0.xml
@@ -9,7 +9,6 @@
   <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="" label="keystore" num_partition_sectors="128" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="512.0" sparse="false" start_byte_hex="0x2308000" start_sector="8968"/>
   <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="" label="frp" num_partition_sectors="128" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="512.0" sparse="false" start_byte_hex="0x2388000" start_sector="9096"/>
   <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="" label="minidump" num_partition_sectors="32768" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="131072.0" sparse="false" start_byte_hex="0x2408000" start_sector="9224"/>
-  <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="cust.img" label="cust" num_partition_sectors="524288" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="2097152.0" sparse="true" start_byte_hex="0xa408000" start_sector="41992"/>
   <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="super.img" label="super" num_partition_sectors="2228224" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="8912896.0" sparse="true" start_byte_hex="0x8a408000" start_sector="566280"/>
   <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="vbmeta_system.img" label="vbmeta_system_a" num_partition_sectors="16" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="64.0" sparse="false" start_byte_hex="0x2aa408000" start_sector="2794504"/>
   <program SECTOR_SIZE_IN_BYTES="4096" file_sector_offset="0" filename="vbmeta_system.img" label="vbmeta_system_b" num_partition_sectors="16" partofsingleimage="false" physical_partition_number="0" readbackverify="false" size_in_KB="64.0" sparse="false" start_byte_hex="0x2aa418000" start_sector="2794520"/>
